tbx/project_styleguide/templates/patterns/molecules/streamfield/blocks/numeric_stats_group_block.html

```diff
@@ -1,15 +1,12 @@
 {% load wagtailcore_tags wagtailimages_tags %}
-<div class="grid__division-signpost">
-    {# Section heading #}
+<div class="grid__stats-group stats-group">
     {% if value.title %}
-        <h2 class="heading heading--two">{{ value.title }}</h2>
+    {% include "patterns/atoms/motif-heading/motif-heading.html" with heading_level=2 heading="Lighting little fires" classes="motif-heading motif-heading--two-c motif-heading--static stats-group__heading" %}
```

Comment:
Question: Should the `heading` be set from a variable or will that be implemented as part of https://torchbox.atlassian.net/browse/TWE-28?